consensus/src/dag/dag_handler.rs

```diff
@@ -108,65 +132,62 @@ impl NetworkHandler {
         }
     }
 
-    async fn process_rpc(
+    async fn process_verified_message(
         &mut self,
-        rpc_request: IncomingDAGRequest,
+        dag_message_result: anyhow::Result<DAGMessage>,
```

Comment:
we can probably discard the error in the verify task instead of passing it here

Reply:
We cast this error and send it back to the caller, so the sender knows if its a verification error.